src/components/SearchBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "@emotion/styled";
+import FormInput from "./FormInput";
+import LocationIcon from "../assets/images/icons/icon-location.svg";
+import SearchIcon from "../assets/images/icons/icon-search.svg";
+
+const Row = styled.div`
+  display: flex;
+  padding: ${(props) => props.theme.layout.space400};
+`;
+
+const Column = styled.div`
+  flex: ${(props) => props.flex || 1};
+  padding: ${(props) => props.theme.layout.space100};
+  border: 1px solid #ccc;
+  border-top-left-radius: ${(props) => props.borderTopLeftRadius || "0"};
+  border-bottom-left-radius: ${(props) => props.borderBottomLeftRadius || "0"};
+  border-top-right-radius: ${(props) => props.borderTopRightRadius || "0"};
+  border-bottom-right-radius: ${(props) =>
+    props.borderBottomRightRadius || "0"};
+  border-left: ${(props) => props.borderLeft || "1px solid #ccc"};
+  border-right: ${(props) => props.borderRight || "1px solid #ccc"};
+  display: flex;
+`;
+
+const SearchBar = () => {
+  return (
+    <Row>
+      <Column flex={1.5} borderTopLeftRadius="8px" borderBottomLeftRadius="8px">
+        <FormInput icon={SearchIcon} altText={"Search Icon"} placeholder={"Filter by title, companies, expertise..."}/>
+      </Column>
+      <Column flex={1.2} borderLeft="0" borderRight="0">
+        <FormInput icon={LocationIcon} altText={"Location Icon"}  placeholder={"Filter by location..."} />
+      </Column>
+      <Column borderTopRightRadius="8px" borderBottomRightRadius="8px"></Column>
```

Comment:
Nitpick: Can you add a `@TODO` comment here to note that we will need to add the Checkbox and Button components once they are complete? I'll create a separate ticket for you to complete next sprint!